src/hal/src/acceleration_structure.rs

```diff
@@ -0,0 +1,224 @@
+//! TODO docs
+
+// TODO remove and add docs
+#![allow(missing_docs)]
+
+use std::ops::Range;
+
+use crate::{buffer::Offset, format::Format, Backend, IndexType};
+
+/// Denotes the type of acceleration structure.
+#[derive(Debug)]
+pub enum AccelerationStructureType {
```

Comment:
if we are in the `acceleration_structure` module, we don't have to prepend all the types with `AcceleratedStructure` prefix. I know it can be useful for doing things like `use acceleration_structure::{A, B}` without renaming `A` and `B`, but so far in gfx codebase we preferred a style where the users would just do `use hal::acceleration_structure as accel` and access the contents as `accel::Type`, `accel::Descriptor`, etc.
So I think these prefixes should be removed.

Reply:
I like this style you've described, I did notice (in the Vulkan API) my eyes start to glaze over with everything with the same (long) prefix.